R/sensitivity-analysis-task.R

```diff
@@ -10,23 +10,26 @@ SensitivityAnalysisTask <- R6::R6Class(
     variationRange = NULL,
     numberOfCores = NULL,
     quantileVec = NULL,
+    parametersToPerturb = NULL,
 
     #' @description
     #' Create a `SensitivityAnalysisTask` object
     #' @param variationRange variation range for sensitivity analysis
     #' @param numberOfCores number of cores for parallel computation
-    #' @param generatedResultFileNames name of files where PK parameters are saved
+    #' @param parametersToPerturb parameters to perturb when performing sensitivity analysis
     #' @param ... parameters inherited from R6 class `Task` object
     #' @return A new `SensitivityAnalysisTask` object
     initialize = function(

```

Comment:
It would be good to add validations here as we do in core and tlf to ensure that parameters hvea the right type
Also `parametersToPerturb `  Are those instances of the `PArameter` class in core?

Reply:
`parametersToPerturb` is of the same format as the output of `ospsuite::potentialVariableParameterPathsFor()`, ie a list of strings of paths.  The two are handled the same way.

Reply:
Ok shouldn't we rename it to avoid confusion? At least the doc on the parameter should clarify this

Reply:
to `variableParameterPaths` ok?